adodb.inc.php

```diff
@@ -5363,7 +5373,8 @@ function ADOLoadDB($dbType) {
 	/**
 	 * Load the code for a specific database driver. Private function. Do not use.
 	 */
-	function ADOLoadCode($dbType) {
+	function ADOLoadCode($dbType,$pdoExtension=false) {
```

Comment:
Why add a $pdoExtension parameter ? Wouldn't it be simpler to just build and pass the appropriate `pdo_XXX` string from ADONewConnection() ?